test/helpers/utils.ts

```diff
@@ -481,3 +483,16 @@ export const serializeToObject = (solidityValue: unknown) => {
   }
   return solidityValue;
 };
+
+export const getDeltaStep = (chainId: number) => {
```

Comment:
The delta step should really be different for each asset, rather than chains.

These are what we're doing for the assets:
```
ETH - 100
WBTC - 1000
AAVE - 10
AVAX - 5
```

Reply:
Yeah good point, fixing.

Reply:
Fixed.